lib/ecto/changeset.ex

```diff
@@ -719,6 +723,7 @@ defmodule Ecto.Changeset do
         {changeset, false}
       end
 
+    forced_update? = Keyword.get(opts, :force_update_on_change, true)
```

Comment:
You can pass the opts to `force_update` and read it there. There is no need to read it before if we are not going to use it. :)